extensions/stats/plugin.h

```diff
@@ -397,9 +406,27 @@ class PluginContext : public Context {
       : Context(id, root),
         upstream_closed_(false),
         downstream_closed_(false),
-        tcp_reported_(false) {}
+        tcp_reported_(false),
```

Comment:
Seems like this is not used. Remove this?

Reply:
It is getting used to call logTCP at the end of both downstreamConnectionClose and UpstreamConnectionClose... 
can change name to tcp_closed_ now I think...